hack/tools.go

```diff
@@ -26,7 +26,6 @@ import (
 	_ "github.com/gardener/gardener/hack/api-reference/template"
 
 	_ "github.com/ahmetb/gen-crd-api-reference-docs"
-	_ "github.com/golang/mock/mockgen"
```

Comment:
I see the extension needs the `mockgen` binary as it generates mocks. See https://github.com/gardener/gardener-extension-provider-aws/blob/0a7ca2db2e57b2106f239c816b420c6a62395cb3/pkg/aws/client/mock/doc.go#L15.

How the mockgen binary get's now built under `hack/tools/bin`? Is g/g's hack/tools.go get used?

I was thinking that when a project needs such a binary, the needed import needs to be added such as:
```
	_ "go.uber.org/mock/mockgen"
```

Reply:
> How the mockgen binary get's now built under hack/tools/bin? Is g/g's hack/tools.go get used?

Mostly yes.